application/cms/markdown.py

```diff
@@ -18,9 +18,17 @@ def run(self, root):
             ol.set("class", "govuk-list govuk-list--number eff-list--sparse")
 
 
+class MarkdownCleanerProcessor(Treeprocessor):
+    def run(self, root):
+        for a in root.iter("a"):
+            if "javascript" in a.get("href"):
```

Comment:
Feels like this might false-positive on an href like `http://mysite.com/about/javascript`?